docs/components/InfoPair.js

```diff
@@ -2,10 +2,11 @@ export default {
   header: 'Info Pair',
   description: 'This is a customizable info pair component.',
   snippet:
-        `<ao-info-pair :label="label">
+        `<ao-info-pair :label="label" :label-html="labelHtml">
   {{ info }}
 </ao-info-pair>`,
   apiRows: [
-    { name: 'label', type: 'String, required', default: '-', description: 'Label for info pair.' }
+    { name: 'label', type: 'String, required', default: '-', description: 'Label for info pair.' },
+    { name: 'label-html', type: 'String, required', default: '-', description: 'HTML label for info pair.' }
```

Comment:
both props need to not include the `required` since they're both optional in the docs

Reply:
documentation updated